src/search&currentWeather/CurrentWeatherAside.jsx

```diff
@@ -0,0 +1,39 @@
+const CurrentWeatherAside = ({ currentWeather, units }) => {
+  return (
+    <div>
+      <div className="weather-image-container">
+        <img
+          className="weather-image"
+          src={` https://openweathermap.org/img/wn/${currentWeather.weather[0].icon}@4x.png`}
+          alt={`${currentWeather.weather[0].description}`}
+        />
+      </div>
+      <div className="current-weather-container">
+        <p className="current-weather">
+          {Math.round(currentWeather.main.temp)}
+          <span className="current-weather-unit">
+            {units === "metric" ? "°C" : "°F"}
```

Comment:
This logic of rendering is also duplicated. Consider creating a separate component